### PR TITLE
rename tables added

### PR DESCRIPTION
**Pull request by saravanpa-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/6259

*Description of changes:*

changes for renaming the tables to plural

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: pay-api/migrations/versions/b6e28faea978_rename_tables_to_plurals.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+# revision identifiers, used by Alembic.
+revision = 'b6e28faea978'
+down_revision = '90afcc0d6d9f'
+branch_labels = None
+depends_on = None
+
+###
+# get_pk_constraint
+
+table_mapping = {'bcol_payment_account ': 'bcol_payment_accounts',
```
Comment (sumesh-aot) [inline pay-api/migrations/versions/b6e28faea978_rename_tables_to_plurals.py:28]: bcol_payment_account, credit_payment_account and internal_payment_account are not in use, we kept it there as a backup.

File: pay-api/migrations/versions/b6e28faea978_rename_tables_to_plurals.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+                 'payment': 'payments',
+                 'payment_account': 'payment_accounts',
+                 'payment_line_item': 'payment_line_items',
+                 'payment_method': 'payment_methods',
+                 'payment_status_code': 'payment_status_codes',
+                 'payment_system': 'payment_systems',
+                 'payment_transaction': 'payment_transactions',
+                 'receipt': 'receipts',
+                 'statement': 'statements',
+                 'transaction': 'transactions',
```
Comment (sumesh-aot) [inline pay-api/migrations/versions/b6e28faea978_rename_tables_to_plurals.py:61]: transaction table I think is created by continuum, so we cannot rename them ?

**Review by sumesh-aot — APPROVED** [review]
Please see the comments

File: pay-api/migrations/versions/b6e28faea978_rename_tables_to_plurals.py (hunk 2 above)
Comment (saravanpa-aot) [reply pay-api/migrations/versions/b6e28faea978_rename_tables_to_plurals.py:61]: will remove it.

**Pull request merged**